tests/testthat/test-qenv_within.R

```diff
@@ -60,6 +60,8 @@ testthat::test_that("within.qenv renturns a `qenv` where `@env` is a deep copy o
   qq <- within(q, {})
   testthat::expect_equal(q@env, qq@env)
   testthat::expect_false(identical(q@env, qq@env))
+  # TODO: fix
+  # dunno what's going on yet
```

Comment:
In eval_code you've introduced following line which exists before environment is cloned.
```r
  if (length(parsed_code) == 0) {
    return(object)
  }
```

Reply:
Ah, thanks for the info!

Reply:
86e5d9588b732e4a4ee635c08db83dc645f73235